ha/core/event_analyzer/parser/parser.py

```diff
@@ -165,6 +165,9 @@ def __init__(self):
         """
         super(ClusterResourceParser, self).__init__()
         ConfigManager.init("event_analyzer")
+        self.cluster_id = Conf.get(const.HA_GLOBAL_INDEX, f"COMMON_CONFIG{_DELIM}cluster_id")
+        self.site_id = Conf.get(const.HA_GLOBAL_INDEX, f"COMMON_CONFIG{_DELIM}site_id")
```

Comment:
Any particular reason to move these inits here from parse_event ?

Reply:
Because its a known initialization. So, I thought we can move this to init. No major reason as such. 
As class instantiation will also be happening repeatedly as and when event will be received. So, I think doesn't  matter if we do it in parse_event or init. But I wanted to make it available class available and we already know the value. Hence, moved it to init()